wpilibc/src/main/native/include/frc/ResistanceCalculator.h

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#pragma once
+
+#include <units/current.h>
+#include <units/impedance.h>
+#include <units/power.h>
+#include <units/voltage.h>
+#include <wpi/circular_buffer.h>
+
+namespace frc {
+
+using ampere_squared_t = units::unit_t<units::squared<units::current::amperes>>;
+using volt_squared_t = units::unit_t<units::squared<units::voltage::volts>>;
```

Comment:
Move these to the top of the private section of the class to avoid namespace pollution.